### ISSUE TITLE
“Expand” the UI to take full advice of the strewn 

### ISSUE THREAD

**Issue by MIfoodie** [elk-zone/elk#1866]
![852A53CC-CB48-4239-8095-E82DF91319D6](https://user-images.githubusercontent.com/94649676/223598542-f2d24ff9-8566-436c-87cd-82f34f9621a5.jpeg)

This is a little mock-up I did of where you could resize and reposition UI elements to make better use of the screen

I hope it’s not too messy :) 🖍️

**Comment by ayoayco** [comment]
One accessibility consideration here. Expanding the status/post card will mean more characters to read per line. There's an optimum number of characters per line that we keep to make it easy on the eyes to read.

**Comment by patak-dev** [comment]
There have been several others asking for this. We can leave it open for people to discuss but I think it is a really hard feature to implement and maintain. The issue is that the current layout for posts expects you to have a width of 600px (or a bit less). Expansions, attachments, polls, and other features are implemented with this in mind. I think a possible way to keep this simple would be to allow for wider posts (let's say 1200px), and then when there is a post card use two columns with the card on the right and the post text on the left. I don't know how good this would look but at least it is something that feels doable.

**Comment by MIfoodie** [comment]
I found this extension thing that uses the whole screen for mastodon and it looks pretty nice. https://codeberg.org/Freeplay/Mastodon-Modern

**Comment by ayoayco** [comment]
I think this should also be left for user-land solutions like a browser extension 😄

**Comment by bhcompy** [comment]
Guess there's no movement on this?  Kind of looks comical on a 1440p monitor.  It's so sad and lonely with all that padding

**Comment by Christopher-Hayes** [comment]
I don't think it's unreasonable to want to use that space for sidebars and other content, but expanding the posts to be basically fullscreen would definitely harm readability, not to mention WCAG. This might not be obvious on a smaller laptop screen, but on 1440p this would be a serious problem.

For example, this is how it would look on 1440p:
![image](https://github.com/elk-zone/elk/assets/6013871/8457cb50-5dfa-4f4f-b727-3045adbdac1c)

However, if Elk goes through a redesign, a slightly wider post layout could still totally work, just probably not fullscreen. This would likely need larger font sizes and padding to make it all work. Elk is already using `rem` on fonts, so font-size on larger screens could be tweaked pretty easily, for example:

![image](https://github.com/elk-zone/elk/assets/6013871/cfb989fd-28bd-46f0-8df4-d646e646ab56)

**Comment by bhcompy** [comment]
I understand what you're saying, but I happen to like it.  I mean I guess I can use tampermonkey to fix it, as I've found or created solutions for other sites that have the same limitations.  Choice is nice.  Luckily, browsers aren't completely constrained

**Comment by nileshtrivedi** [comment]
Whalebird has a Slack/Discord like interface which is really great, but it works for them because Whalebird is a desktop-only client. 

![image](https://github.com/user-attachments/assets/50541def-29fd-4029-9ada-020b19fe2d1a)

I am interested in showing the Followed Hashtags in the Elk sidebar if on large screens (again, like Slack/Discord): 

![image](https://github.com/user-attachments/assets/10e5545b-6762-42dc-b773-43b43147b50c)

Will try to accomplish this via a tampermonkey script first.

**Comment by nileshtrivedi** [comment]
I implemented the above feature (followed hashtags in the sidebar for Slack/Discord-like topic-wise browsing/discovery) and in Whalebird desktop client and love it! Would love to see this implemented in Elk too.

<img width="1142" alt="image" src="https://github.com/user-attachments/assets/f2e949d3-da15-434a-a3d9-e2a194beb10d">

**Comment by rimu** [comment]
I can see why you might want to maintain the current design on a desktop but on a mobile it's not great. I have highlighted wasted space where lines of text would benefit from being wider in this screenshot.

<img width="720" height="1600" alt="Image" src="https://github.com/user-attachments/assets/94b23688-a400-43ad-8662-e0f56b13a7b0" />
